microsoftgraph/microsoft-graph-explorer-v4
# Task: autocomplete hover styling

## Overview

Show the user which autocomplete options is hovered on.

Fixes #725 

### Demo

Hovering over assignedLicenses
![image](https://user-images.githubusercontent.com/58787602/104914924-e49bfe80-59a0-11eb-9726-54357bee42c3.png)


## Testing Instructions

* Trigger autocomplete
* Hover over the available options
* Notice a change in background colour

* change theme and notice the effect is as expected
